sdk/instrumentation/google.golang.org/grpc/status_text.go

```diff
@@ -0,0 +1,66 @@
+package grpc
+
+func StatusText(code int) string {
+	switch code {
+	case 400:
+		return "Bad Request"
+	case 401:
+		return "Unauthorized"
+	case 402:
```

Comment:
Nit: Also maybe we can turn this into a map variable and use the map instead of a switch. I think it's faster? If not found in map we can return "Request Error". To be honest I don't know what's faster between a map and a switch :)

Reply:
Thanks for sharing these.